src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -484,8 +488,14 @@ internal void CloseWindow()
             // RequestUpdateLoadBarStatus event needs to be unsubscribed when the splash screen window is closed, to avoid populating the info on splash screen.
             else
             {
-                this.Close();
-                viewModel?.Model.ShutDown(false);
+                Close();
+                if (viewModel != null)
+                {
+                    viewModel.RequestClose -= SplashScreenRequestClose;
```

Comment:
the `viewModel?.Model.ShutDown(false)` is not more necessary, right?

Reply:
as far as I can tell, it's not necessary because under normal circumstances calling close on the DynamoView (even if it's hidden) will still end up calling shutdown on the viewmodel.